src/crypto/ecc_curve.rs

```diff
@@ -1,6 +1,7 @@
 use crate::crypto::public_key::PublicKeyAlgorithm;
+use asn1_rs::FromDer;
```

Comment:
I'd prefer to use https://docs.rs/der/ from rustcrypto

Reply:
I pushed a variant that uses the `der` crate. However, that crate seems to produce a differing String format, and I don't have a good enough grasp of OIDs and ASN.1 to make sense of what's going wrong there:

```
test crypto::ecc_curve::tests::test_unknown_ecc_curve ... FAILED

failures:

---- crypto::ecc_curve::tests::test_unknown_ecc_curve stdout ----
thread 'crypto::ecc_curve::tests::test_unknown_ecc_curve' panicked at 'assertion failed: `(left == right)`

Diff < left / right > :
<0.6.9.43.36.3.3.2.8.1.1.1
>1.3.36.3.3.2.8.1.1.1

', src/crypto/ecc_curve.rs:214:9
```

Reply:
This PR now uses the `der` crate for OID handling.

The test issue is resolved, based on Tony's input, below (my test was wrong).

Reply:
Also if you just need OID support, there's the `const-oid` crate

Reply:
Thanks! For the specific use case in this PR, `const-oid` is sufficient.
I'll switch to that, for now, until more features are required in rpgp.